src/services/AuthorizationService.php

```diff
@@ -86,7 +86,13 @@ public function __construct($authorizationUrl)
      */
     public function getAuthorization()
     {
-        list($code, $header , $body) = RequestHelper::get($this->authorizationUrl);
+        $jwk = OpenSSLHelper::generateJWSOfKid(
```

Comment:
Do we want to handle the following exceptions now as well?

- \stonemax\acme2\exceptions\NonceException

Reply:
I don't think there's a need, if an exception did occur and was handled the request would fail anyway, and the thrown exception would be logged to Raygun.

Reply:
Right. I just wanted to make sure all exceptions are handle,... I guess that's an ok process to go on as well.